src/main/java/edit_user_public_profile_screens/editUserPublicProfileScreen.java

```diff
@@ -0,0 +1,77 @@
+package edit_user_public_profile_screens;
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class editUserPublicProfileScreen extends JFrame implements ActionListener{
+
+    public editUserPublicProfileScreen() {
+        this.setVisible(true);
+        this.setSize(500, 500);
+
+        JPanel main = new JPanel();
+        main.setLayout(new BorderLayout());
+
+        JPanel menubar = new JPanel();
+        JLabel title = new JLabel("Edit User Public Profile");
+        menubar.add(title);
+
+        JButton exit = new JButton("Exit");
+        menubar.add(exit);
+
+        main.add(menubar, BorderLayout.NORTH);
+
+        /*Adding options*/
+        JPanel preferences = new JPanel();
+        preferences.setLayout(new GridLayout(0, 2));
+
+        JLabel locationLabel = new JLabel("Location: ");
+        JPanel locationOptions = new JPanel();
+        JCheckBox locationOption1 = new JCheckBox("Online");
+        JCheckBox locationOption2 = new JCheckBox("In-person");
+        locationOptions.add(locationOption1);
+        locationOptions.add(locationOption2);
+
+        JLabel meetingTimeLabel = new JLabel("Meeting Time: ");
+        JPanel meetingTimeOptions = new JPanel();
+        JCheckBox meetingTimeOption1 = new JCheckBox("Morning");
+        JCheckBox meetingTimeOption2 = new JCheckBox("Afternoon");
+        JCheckBox meetingTimeOption3 = new JCheckBox("Evening");
```

Comment:
Do you want to keep meeting time options as "morning, afternoon, or evening" or should we do days of the week? We can discuss this at our meeting later today!